### DISCUSSION TITLE
Project groups

### DISCUSSION THREAD

**Discussion by SVillette** [mosparo/mosparo#231] [Ideas]
Hi,

Thank you for developping Mosparo. I really like the way this project has been created. 

In my project, Mosparo is implemented only on the registration page. So I created 2 projects in Mosparo. One for production and one for pre-production. But this implementation means that for each page I need to add Mosparo, I have to create two projects. At the end, I think it can be messy and project groups could resolve this issue by dividing the projects listing into groups.

What do you think ?

**Comment by zepich** [comment]
Hi @SVillette 

Thank you very much for your feedback! It's fantastic to hear that!

And thank you very much for your great idea! I don't know why I didn't have this idea, but if you have a lot of projects, grouping them is absolutely a required feature.

We will add such a feature with version 1.3 of mosparo, which will be released in the second half of 2024 (probably August/September).

Please let me know if you have any other suggestions!

Kind regards,

zepich

**Reply by SVillette** [reply]
Thanks for you fast reply, that's great to hear !

**Comment by zepich** [comment]
Added with v1.3
